test/compiler.test.js

```diff
@@ -662,6 +693,72 @@ describe('compiler', function() {
       });
     });
 
+    it('resolves relative path in `modelSources` option', function() {
+      appdir.createConfigFilesSync({}, {}, {
+        Car: { dataSource: 'db' }
+      });
+      appdir.writeConfigFileSync('custom-models/car.json', { name: 'Car' });
+      var appJS = appdir.writeFileSync('custom-models/car.js', '');
+
+      var instructions = boot.compile({
+        appRootDir: appdir.PATH,
+        modelSources: ['./custom-models']
+      });
+
+      expect(instructions.models).to.have.length(1);
+      expect(instructions.models[0].sourceFile).to.equal(appJS);
+    });
+
+    it('resolves non-relative path in `modelSources` option', function() {
+      appdir.createConfigFilesSync({}, {}, {
+        Car: { dataSource: 'db' }
+      });
+      appdir.writeConfigFileSync('custom-models/car.json', { name: 'Car' });
+      var appJS = appdir.writeFileSync('custom-models/car.js', '');
+
+      var instructions = boot.compile({
+        appRootDir: appdir.PATH,
+        modelSources: ['custom-models']
```

Comment:
IMO, the primary purpose of this variant is to load `node_modules/custom-models` instead of `appRootDir + '/custom-models`.

When the directory does not exist in `node_modules`, then I guess it's ok to try to resolve the path as relative to `appRootDir`, although I am also ok with the solution where non-relative paths are always resolved as a module path in one of `node_modules` folders.

What I would like you to do: Either rework this test to lookup the files in `node_modules/custom-models/car*`, or add a new test that will verify this scenario.